packages/server/src/mail/user-subscriptions.entity.ts

```diff
@@ -0,0 +1,30 @@
+import {
+  Entity,
+  PrimaryGeneratedColumn,
+  ManyToOne,
+  BaseEntity,
+  Column,
+} from 'typeorm';
+import { UserModel } from '../profile/user.entity';
+import { MailServiceModel } from './mail-services.entity';
+
+@Entity('user_subscriptions')
+export class UserSubscriptionModel extends BaseEntity {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column()
+  serviceId: number;
+
+  @Column()
+  userId: number;
+
+  @Column()
+  isSubscribed: boolean;
+
+  @ManyToOne(() => UserModel, (user) => user.subscriptions)
+  user: UserModel;
+
+  @ManyToOne(() => MailServiceModel, (service) => service.subscriptions)
+  service: MailServiceModel;
```

Comment:
add onDelete: 'CASCADE'. You will likely need to delete your migration file and then generate a new one